CMakeLists.txt

```diff
@@ -36,6 +36,48 @@ mark_as_advanced(MATERIALX_PYTHON_EXECUTABLE)
 mark_as_advanced(MATERIALX_PYTHON_OCIO_DIR)
 mark_as_advanced(MATERIALX_PYTHON_PYBIND11_DIR)
 
+# Validation options for compiling and rendering generated shaders
+option(MATERIALX_BUILD_RENDER "Build the MaterialX Render module." ON)
```

Comment:
Since MaterialXRender, MaterialXRenderGlsl, and MaterialXRenderOsl have no external dependencies or known platform limitations, I would actually recommend that we remove their build flags, and allow them to be built in all configurations.  (It's still a great idea to have a MATERIALX_TEST_RENDER flag, since access to a GPU is not universal.)